### PR TITLE
Add more information to the Response class

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
Let me know if something is missing.

It uses infix functions because pretty code.

### CODE REVIEW

**Review by h1dden-da3m0n — APPROVED** [review]

File: jellyfin-api/src/main/kotlin/org/jellyfin/apiclient/api/client/Response.kt
Comment (Maxr1998) [inline jellyfin-api/src/main/kotlin/org/jellyfin/apiclient/api/client/Response.kt:69]: Are those really that necessary? I'm not sure of the benefit these functions bring..

Comment (nielsvanvelzen) [reply jellyfin-api/src/main/kotlin/org/jellyfin/apiclient/api/client/Response.kt:69]: I've actually used `response isStatus 200` in an upcoming PR

Comment (Maxr1998) [reply jellyfin-api/src/main/kotlin/org/jellyfin/apiclient/api/client/Response.kt:69]: But is that actually so much better than a `response.status == 200`?

Comment (nielsvanvelzen) [reply jellyfin-api/src/main/kotlin/org/jellyfin/apiclient/api/client/Response.kt:69]: Hmm.. actually you're right. The status value was private at first but since it's public anyway we don't really need these functions.

**Review by Maxr1998 — APPROVED** [review]

**Comment by Maxr1998** [comment]
@nielsvanvelzen should we wait for a review from @thornbill or can I (squash) merge it already?

**Review by h1dden-da3m0n — APPROVED** [review]

**Pull request merged**
